composer/algorithms/channels_last/channels_last.py

```diff
@@ -30,6 +30,18 @@ def apply_channels_last(model: torch.nn.Module) -> None:
     model.to(memory_format=torch.channels_last)  # type: ignore
 
 
+def remove_channels_last(model: torch.nn.Module) -> None:
```

Comment:
A few considerations here:
* should this be a functional API, or a method of the `Algorithm` class? e.g. in case some algorithms may need to store some state to know how to revert
* I am assuming the remove should only apply changes that were done on `INIT`? And assume that subsequently, the algorithm itself is removed from running?

Reply:
Lets discuss offline -- I think we might need to rework parts of the Algorithm class to be reversible

Reply:
The former I think the answer is yes.

The latter also yes